src/auth/src/credentials/user_account.rs

```diff
@@ -188,26 +194,101 @@ impl Builder {
         self
     }
 
-    /// Returns a [Credentials] instance with the configured settings.
+    /// Configure the retry policy for fetching tokens.
     ///
-    /// # Errors
+    /// The retry policy controls how to handle retries, and sets limits on
+    /// the number of attempts or the total time spent retrying.
     ///
-    /// Returns a [CredentialsError] if the `authorized_user`
-    /// provided to [`Builder::new`] cannot be successfully deserialized into the
-    /// expected format. This typically happens if the JSON value is malformed or
-    /// missing required fields. For more information, on how to generate
-    /// `authorized_user` json, consult the relevant section in the
-    /// [application-default credentials] guide.
+    /// ```
+    /// # use google_cloud_auth::credentials::user_account::Builder;
+    /// # use gax::retry_policy;
+    /// # use gax::retry_policy::RetryPolicyExt;
+    /// # tokio_test::block_on(async {
+    /// let authorized_user = serde_json::json!({
+    ///     "client_id": "YOUR_CLIENT_ID.apps.googleusercontent.com",
+    ///     "client_secret": "YOUR_CLIENT_SECRET",
+    ///     "refresh_token": "YOUR_REFRESH_TOKEN",
+    ///     "type": "authorized_user",
+    /// });
+    /// let credentials = Builder::new(authorized_user)
+    ///     .with_retry_policy(retry_policy::AlwaysRetry.with_attempt_limit(3))
+    ///     .build();
+    /// # });
+    /// ```
+    pub fn with_retry_policy<V: Into<RetryPolicyArg>>(mut self, v: V) -> Self {
+        self.retry_builder = self.retry_builder.with_retry_policy(v.into());
+        self
+    }
+
+    /// Configure the retry backoff policy.
     ///
-    /// [application-default credentials]: https://cloud.google.com/docs/authentication/application-default-credentials
-    pub fn build(self) -> BuildResult<Credentials> {
+    /// The backoff policy controls how long to wait in between retry attempts.
+    ///
+    /// ```
+    /// # use google_cloud_auth::credentials::user_account::Builder;
+    /// # use gax::exponential_backoff::ExponentialBackoffBuilder;
+    /// # use std::time::Duration;
+    /// # tokio_test::block_on(async {
+    /// let authorized_user = serde_json::json!({
+    ///     "client_id": "YOUR_CLIENT_ID.apps.googleusercontent.com",
+    ///     "client_secret": "YOUR_CLIENT_SECRET",
+    ///     "refresh_token": "YOUR_REFRESH_TOKEN",
+    ///     "type": "authorized_user",
+    /// });
+    /// let policy = ExponentialBackoffBuilder::new()
+    ///     .with_initial_delay(Duration::from_millis(100))
+    ///     .with_maximum_delay(Duration::from_secs(5))
+    ///     .with_scaling(4.0)
+    ///     .build().expect("well-known policy values should succeed");
```

Comment:
maybe use `ExponentialBackoffBuilder::default()` so you can skip the `.expect()` and the code is shorter too.